File: Python/Program/Keyboard.py
from Komponen import Komponen

class Keyboard(Komponen):
    def __init__(self, jenis, backlight, merk, nama, jenis_komponen):
        super().__init__(merk, nama, jenis_komponen)
        self.jenis = jenis
        self.backlight = backlight

    def get_jenis(self):
        return self.jenis

    def get_backlight(self):
        return self.backlight

File: C++/Program/Keyboard.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Keyboard : public Komponen
{
private:
    string jenis;
    string backlight;
public:
    Keyboard()
    {

    } 

    Keyboard(string jenis, string backlight, string nama, string merk) : Komponen(nama, merk)
    {
        this->jenis = jenis;
        this->backlight = backlight;
    }

    void setJenis(string jenis)
    {
        this->jenis = jenis;
    }

    string getJenis()
    {
        return jenis;
    }

    void setBacklight(string backlight)
    {
        this->backlight = backlight;
    }

    string getBacklight()
    {
        return backlight;
    }

    ~Keyboard() {}
};